LibraBFT/Base/KVMap.agda

```diff
@@ -59,8 +59,16 @@ module LibraBFT.Base.KVMap  where
                   → lookup k kvm ≢ nothing
                   → KVMap Key Val
 
+   kvm-insert-Haskell
+                  : (k : Key)(v : Val)(kvm : KVMap Key Val)
+                  → KVMap Key Val
+
+   kvm-member     : (k : Key) (kvm : KVMap Key Val)
```

Comment:
This could be defined as \ k -> is-just . lookup k

Reply:
I am going to deal with KVMap changes in an independent pull request.